src/pages/Service.js

```diff
@@ -21,7 +21,7 @@ class Service extends Component {
 
   componentDidMount() {
     axios
-      .get(`${process.env.REACT_APP_CMS_ENDPOINT}/pages/${this.props.match.params.id}?fields=content,extra_content,theme(text)`)
+      .get(`${process.env.REACT_APP_CMS_ENDPOINT}/pages/${this.props.match.params.id}?fields=content,extra_content,topic(text),locations(location(name,street,city,state,zip,country,hours)),contacts(contact(name,email,phone))`)
```

Comment:
Live preview currently seems very coupled to the service page. Any plans for how we might achieve preview for all pages?
Also, would postMessage work for passing content to the iframe in a cleaner/safer way?
https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage